### PR TITLE
Updates the extension to work with new app-style redmart website. Added functionality to read other units (g, kg, ml, L, per pack) and also process offers (e.g. Buy 2 for $4.85).

### PR DESCRIPTION
**Pull request by lihweilihwei**

### CODE REVIEW

**Comment by lihweilihwei** [comment]
Would like to add that I made the extension work upon clicking the extension button, or can be set to a keyboard shortcut by the user (suggested is Ctrl+1). I also reduced the time delay for processing prices to 0.5 seconds.

File: manifest.json [hunk 1]
```diff
@@ -3,20 +3,26 @@
   "version": "0.0.1",
```
Comment (lologhi) [inline manifest.json:3]: Good news! For your update to be seen by Google, you just need to update the version number!

> The Chrome Browser periodically checks for new versions of installed extensions and updates them without user intervention.

[source](https://developer.chrome.com/extensions/hosting#updating)

File: manifest.json [hunk 2]
```diff
@@ -3,20 +3,26 @@
...
   "manifest_version": 2,
   "description": "Because it's the only way to compare prices",
-  "homepage_url": "https://github.com/lologhi/redmart-per-kilo",
+  "permissions": ["activeTab", "declarativeContent", "*://*/*"],
+  "homepage_url": "https://github.com/lihweilihwei/redmart-per-kilo",
```
Comment (lologhi) [inline manifest.json:7]: Maybe not needed to change the `homepage_url` to your fork now that you can merge ;-)

File: manifest.json [hunk 3]
```diff
@@ -3,20 +3,26 @@
...
+	"scripts": ["background.js"],
+    "persistent": true
+  },
+  "browser_action": {
+    "default_icon": "icons/icon128.png"
+  },
+  "commands": {
+	"_execute_browser_action": {
+		"suggested_key": {
+			"default": "Ctrl+1",
```
Comment (lologhi) [inline manifest.json:18]: Maybe add an explanation in the readme.md to explain the shortcuts.

**Review by lologhi — COMMENTED** [review]
A late review :p

File: manifest.json (hunk 1 above)
Comment (lihweilihwei) [reply manifest.json:3]: Got it! I changed the version number to 0.0.2. I'll pack the updated extension too.

Comment (lihweilihwei) [reply manifest.json:3]: I have updated the packed extension (.crx) now it just needs to be published on your developer dashboard.

**Pull request merged**